libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -298,8 +298,12 @@ void elasticsearch_plugin_impl::createBulkLine(const account_transaction_history
 
 void elasticsearch_plugin_impl::prepareBulk(const account_transaction_history_id_type& ath_id)
 {
-   std::string _id = fc::json::to_string(ath_id);
-   prepare = graphene::utilities::createBulk(index_name, bulk_line, _id, 0);
+   const std::string _id = fc::json::to_string(ath_id);
+   fc::mutable_variant_object bulk_header;
+   bulk_header["_index"] = index_name;
+   bulk_header["_type"] = "data";
+   bulk_header["_id"] = fc::to_string(ath_id.space_id) + "." + fc::to_string(ath_id.type_id) + "." + ath_id.instance;
```

Comment:
Why not use something like `fc::to_string(ath)` or even directly use `ath`?

Reply:
yes, i dont like this part. if i do `fc::to_string(ath.id)` it will add double `"` to the final string. if i do  `ath_id` directly the compiler complains that cant conver it. looking for alternatives.

Reply:
Makes sense. Thanks. Current code is fine.